### PR TITLE
Simplified Chinese localization support has been added

### PR DESCRIPTION
**Pull request by cary-rowen**
Simplified Chinese localization support has been added, the 'Switch between top-level windows' command is displayed in the input gesture dialog, and some translation not valid errors have been fixed

### CODE REVIEW

File: addon/globalPlugins/winWizard.py [hunk 1]
```diff
@@ -456,13 +455,10 @@ def terminate(self):
 		del self.hiddenWindowsList
 
 	@scriptHandler.script(
-		description=_(
-			# Translators: Description of the keyboard command
-			# allowing to jump between top-level windows of the current application.
-			"Jumps between top-level windows of the current application."
-		),
+			# Translators: Description of the keyboard command that allowing to jump between top-level windows of the current application.
```
Comment (lukaszgo1) [inline addon/globalPlugins/winWizard.py:458]: Why you've changed this?

File: addon/globalPlugins/winWizard.py [hunk 2]
```diff
@@ -456,13 +455,10 @@ def terminate(self):
...
+		description=_("Jumps between top-level windows of the current application."),
 		gesture="kb:NVDA+windows+Tab",
-		category=SCRCAT_FOCUS,
+#		category=SCRCAT_FOCUS,
```
Comment (lukaszgo1) [inline addon/globalPlugins/winWizard.py:461]: Why this change. I've intentionally placed this script in the system focus category of the input gestures dialog as this is more intuitive that way.

**Review by lukaszgo1 — COMMENTED** [review]
Thanks for your contribution! Unfortunately translations for NVDA add-ons are managed through a separate system which currently does not accept new add-ons. I'm aware that some developers are accepting translation files as a pr against the add-on repository but I'm not going to do that, sorry.
It increases fragmentation and removes main incentive for NV Access to make translation system working as soon as possible please complain to them more people  will do that the better.
I can promise that as soon as translation system is working for new add-ons again winWizard would be  registered in it and then translations to various languages can be added easily.
I'll happily accept the part of this PR which changes some translatable strings in the code when you would explain some changes which I've asked about below.

**Comment by cary-rowen** [comment]
@lukaszgo1 
Hello, the reason for this change is that novices often only look for this feature under the add-on category, and the documentation does not explain this.
Thank you very much for your contribution.

File: addon/globalPlugins/winWizard.py (hunk 1 above)
Comment (lukaszgo1) [reply addon/globalPlugins/winWizard.py:458]: @cary-rowen Again - why this line is changed?

**Comment by cary-rowen** [comment]
Sorry, I think I may have encountered a language barrier. According to your original settings, I hope you can separately explain the category of the gesture in the user documentation.

**Comment by cary-rowen** [comment]
Is the NVDA translation system currently available?

**Comment by lukaszgo1** [comment]
@cary-rowen I've fixed errors in the usages of format function in  master and explained that command for jumping between top-level windows can be found in the System focus category in the readme. Please take a look at the code in master and let me know if it is okay for you. W.R.T NVDA's translation system - no it still does not work. Since as explained above I'm not going to accept translations via git and code changes from the pr are already in master I'm going to close it.

**Comment by cary-rowen** [comment]
I have tested it and it seems to be very good so far, and it works as expected.

**Comment by cary-rowen** [comment]
I saw this add-on on the add-on website, how should I include the simplified Chinese language file at the moment? The download page of the add-on has been translated.

**Comment by lukaszgo1** [comment]
@cary-rowen  WinWizard is now added to the translations system and you should be able to translate it. If there are any problems with this feel free to ping me.

**Comment by cary-rowen** [comment]
@lukaszgo1 
Thank you for your reminder, but I did not see it in zh_Cn\addons.
Can you check it?

Thanks.

**Comment by lukaszgo1** [comment]
the Chinese translations team needs to enable translations for WinWizard in the settings file - for details see here: https://github.com/nvaccess/nvda/wiki/TranslatingAddons

**Comment by cary-rowen** [comment]
Hi @lukaszgo1 

The update of this add-on has been turned on in the 8th line of Settings under zh_CN, but the translation file of the interface is still not received:
 
    "addon.WinWizard": "1",

Submit after opening, then check SVN update again.
